src/components/StakingVaults/WalletDefiEarn.tsx

```diff
@@ -0,0 +1,87 @@
+import type { FlexProps } from '@chakra-ui/react'
+import { Flex, Tab, TabList, TabPanel, TabPanels, Tabs } from '@chakra-ui/react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { ChainDropdown } from 'components/AssetSearch/Chains/ChainDropdown'
+import { selectPortfolioChainIdsSortedFiat } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { GlobalFilter } from './GlobalFilter'
+import type { PositionTableProps } from './PositionTable'
+import { PositionTable } from './PositionTable'
+import type { ProviderTableProps } from './ProviderTable'
+import { WalletProviderTable } from './WalletProviderTable'
+
+type DefiEarnProps = {
+  positionTableProps?: Omit<PositionTableProps, 'searchQuery'>
+  providerTableProps?: Omit<ProviderTableProps, 'searchQuery'>
+  includeEarnBalances?: boolean
+  includeRewardsBalances?: boolean
+  header?: JSX.Element
+} & FlexProps
+
+export const WalletDefiEarn: React.FC<DefiEarnProps> = ({
+  positionTableProps,
+  providerTableProps,
```

Comment:
Why is eslint not complaining at this being unused, you just broke the compiler ser